crates/iota-json-rpc/src/authority_state.rs

```diff
@@ -354,7 +354,6 @@ impl StateRead for AuthorityState {
             .await?)
     }
 
-    #[allow(clippy::type_complexity)]
```

Comment:
Wait I'm confused now, it actually wasn't complaining?

Reply:
I just moved it to the trait

Reply:
Right, thanks